src/tests/helpers/index.ts

```diff
@@ -26,7 +24,10 @@ async function createGroup(
 }
 
 async function createShipment(input: ShipmentCreateInput): Promise<Shipment> {
-  return await Shipment.create(input)
+  return await Shipment.create({
+    ...input,
+    pricing: input.pricing || undefined,
```

Comment:
Question: why is this necessary?

Reply:
😭 because graphql codegen types non-required things as `Thing | null | undefined` and sequelize types non-required things as `Thing | undefined` 😭 😭 😭 😭 😭

Reply:
See commit message